src/javascript/_common/base/gtm.js

```diff
@@ -162,10 +163,55 @@ const GTM = (() => {
         pushDataLayer(gtm_data);
     };
 
+    const pushTransactionData = (response, extra_data = {}) => {
```

Comment:
seems we're only pushing deposit and withdrawal data, not other transactions, perhaps renaming this function or adding a comment to explain that will help